docs/opentelemetry.ext.flask.rst

```diff
@@ -0,0 +1,10 @@
+opentelemetry.ext.flask package
```

Comment:
As an aside, it might be a good idea to document everything that someone has to touch when adding a new package. I may not be fully aware of them myself anymore, and this one is easily missed.